### PR TITLE
Update build version

### PR DESCRIPTION
**Pull request by gnalh**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "trunk-analytics-cli"
 edition = "2021"
-version = "0.4.0"
+version = "0.4.1"
```
Comment (gnalh) [inline Cargo.toml:4]:
```suggestion
version = "0.5.0"
```

**Review by riya-n — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "trunk-analytics-cli"
 edition = "2021"
-version = "0.4.0"
+version = "0.5.0"
```
Comment (gnalh) [inline Cargo.toml:4]: Increase minor version since we are adding new functionality in a backwards compatible way.

Comment (pat-trunk-io) [reply Cargo.toml:4]: Can cargo.toml be generated from git tag?

Comment (gnalh) [reply Cargo.toml:4]: That's a great question, I'm not sure.

**Pull request merged**